Framework/src/EventFile.cxx

```diff
@@ -113,7 +113,13 @@ namespace ldmx {
             parent_->tree_->SetBranchStatus(srule.c_str(),1);
             tree_->SetBranchStatus(srule.c_str(),1);
         } else if( isDrop ){
-            parent_->tree_->SetBranchStatus(srule.c_str(),0);
+	  if (! tree_) { 
+	    //this prevents creating a "ghost" branch in the output tree from the enabled branch in the input tree 
+	    parent_->tree_->SetBranchStatus(srule.c_str(),0);
+	    tree_ = parent_->tree_->CloneTree(0);
+	  }
+	  tree_->SetBranchStatus(srule.c_str(),0);
```

Comment:
This line throws the warning:
```
Error in <TTree::SetBranchAddress>: unknown branch -> DROP_BRANCH_NAME
```
This is because `tree_` was cloned when that branch was turned off. This error can be removed by putting this line in the `else` branch of the above `if` tree so this line is only called when `tree_` is already created.

But I don't see a situation where `tree_` is already created. The drop/keep rules are added to the output Event file before any events are read, this means this code is *always* called before the other location where `tree_` could be cloned (at the beginning of `nextEvent`). Maybe put it in an `else` branch with a comment/warning about this being a weird case?